jac-splice-orc/requirements.txt

```diff
@@ -5,3 +5,4 @@ protobuf==4.23.4
 pluggy==1.2.0
 jaclang
 numpy
+deepmerge==2.0
```

Comment:
Let's implement it ourselves without adding this requirement.

Reply:
actually, we shouldn't try to merge the configs. We should just use the config provided by the user OR the default one, as a whole.